apps/web/pages/api/auth/signup.ts

```diff
@@ -128,6 +114,37 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
     if (team) {
       const teamMetadata = teamMetadataSchema.parse(team?.metadata);
+
+      if (
+        IS_CALCOM &&
+        (!teamMetadata?.isOrganization || (!!team.parentId && !teamMetadata?.isOrganization))
```

Comment:
```suggestion
        (!teamMetadata?.isOrganization)
```

Reason: The second or will never be true 'more' than the first.